ui/src/jobs/details/config/configuration_section/resource_request_table/ResourceRequestConfigTable.js

```diff
@@ -56,6 +56,7 @@ export const ResourceRequestConfigTable = ({
           textStyle="reverse"
           type="responsiveColumn"
           listItems={driver_items}
+          columnWidths={[1, 1]}
```

Comment:
Previously the widths of these were divided evenly (50-50) by default, but this default behaviour no longer exists and we now need to explicitly set `columnWidths={[1, 1]}`.